action.yml

```diff
@@ -0,0 +1,57 @@
+# azure spirng cloud action
+name: 'AzureSpringCloud'
+description: 'Deploy applications to Azure Spring Cloud and manage deployments.'
+inputs: 
+  azure-subscription:
+    description: 'Select the Azure Resource Manager subscription for the deployment.'
+    required: true
+  action:
+    description: 'Action to be performed on Azure Spring Cloud.'
+    required: true
+    default: 'Deploy'
+  service-name:
+    description: 'Select the Azure Spring Cloud service to which to deploy.'
+    required: true
+  app-name:
+    description: 'Select the Azure Spring Cloud app to deploy.'
+    required: true
+  use-staging-deployment:
+    description: "Automatically select the deployment that's set as Staging at the time the task runs."
+    required: true
+    default: true
+  create-new-deployment:
+    description: "Whether to target the deployment that's set as Staging at the time of execution. If unchecked, the 'Deployment Name' setting must be set."
+    required: false
+    default: false
+  deployment-name:
+    description: 'The deployment to which this task will apply. Lowercase letters and numbers only; must start with a letter.'
```

Comment:
I presume `-` is also an allowed character.

Reply:
Copied form Azure DevOps pipeline tasks. If - is allowed, I will modify the description.